### PR TITLE
fix: unresponsive content page

### LINKED ISSUE #4492: Unresponsive Menu on the Contents page
**The Menu on the Contents page is unresponsive for mobile devices**
The Menu is unresponsive to mobile devices. This arises due to the overflow of the Menu component.

**To Reproduce**
This can be seen in changing the device width to 375px or similar.

**Expected behavior**
The Page should be responsive for all device widths.

**Screenshots**
Screenshot 2023-03-10 at 12 02 07 PM

**Software:**

- OS: any
- Browser: any

### PR DESCRIPTION
**Pull request by sudhanshu1309**
The menu for contents page was unresponsive on mobile devices Fixed this by changing the menu overflow to scroll.
closes #4492

### CODE REVIEW

File: news/4492.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The menu for the contents page was unresponsive on mobile devices. Fixed this by changing the menu overflow to scroll.
+In `theme/themes/pastanaga/extras/contents.less` file added a `overflow-x: scroll;` to make overflow content to scroll.  @sudhanshu1309
```
Comment (stevepiercy) [inline news/4492.bugfix:2]:
```suggestion
```

File: news/4492.bugfix [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The menu for the contents page was unresponsive on mobile devices. Fixed this by changing the menu overflow to scroll.
```
Comment (stevepiercy) [inline news/4492.bugfix:1]:
```suggestion
The menu for the contents page was unresponsive on mobile devices. Fixed this by changing the menu overflow to scroll. @sudhanshu1309

```

File: news/4492.bugfix (hunk 1 above)
Comment (stevepiercy) [reply news/4492.bugfix:2]: This is obvious in the code diff.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Simplify the news item. Otherwise LGTM.

Silly mister-roboto. Sorry about that confusion.

**Review by stevepiercy — APPROVED** [review]

**Review by sneridagh — APPROVED** [review]

**Pull request merged**